lib/cdo/aws/rds.rb

```diff
@@ -15,7 +15,7 @@ class RDS
     def self.clone_cluster(
       source_cluster_id: CDO.db_cluster_id,
       clone_cluster_id: "#{source_cluster_id}-clone",
-      instance_type: 'db.r4.large',
+      instance_type: 'db.r5.large',
```

Comment:
Since clones are typically used for testing, how about we default to `db.t3.2xlarge`?

The `t3` instances are priced and configured for occasional workload bursts. We're typically cloning the production cluster, so I'm suggesting the largest t3 instance type, because of the memory (32GB) and compute  (8 vCPUs) resources.

https://aws.amazon.com/rds/instance-types/